docs/tools/cli/service/connection-pool.rst

```diff
@@ -31,17 +31,108 @@ Creates a new :doc:`PgBouncer connection pool </docs/products/postgresql/concept
   * - ``--pool-mode``
     - The :ref:`pool mode <pooling-modes>`. Possible values are ``transaction``, ``session`` and ``statement``
 
-**Example:** In the service ``pg-doc`` Create a new connection pool named ``cp-analytics-it`` for the database ``it-analytics`` with:
+**Example:** In the service ``demo-pg`` Create a new connection pool named ``cp-analytics-it`` for the database ``it-analytics`` with:
 
 * username ``avnadmin``
 * pool-size of ``10`` connections 
 * ``transaction`` pool-mode
 
 ::
 
-  avn service connection-pool-create pg-doc \
+  avn service connection-pool-create demo-pg \
```

Comment:
Interesting, when I'm trying to create  a connection pool with this command I get `ERROR	command failed: Error: {"errors":[{"message":"No valid password exists for user avnadmin.","status":400}],"message":"No valid password exists for user avnadmin."}`.

Reply:
This is weird, I just tried again and the command works as expected, maybe you can try again recreating the bug and open an issue in JIRA?

Reply:
Mystery, can't reproduce the error any more! Will keep an eye if I see it again